apps/python-testsgk/python_testsgk/app.py

```diff
@@ -0,0 +1,104 @@
+from flask import Flask, jsonify, request
+import sqlite3 as sl
+
+
+app = Flask(__name__)
+conn = sl.connect("tasks.db")
+curs = conn.cursor()
+
+stmtCreateDB = (
+    "CREATE TABLE IF NOT EXISTS tasks('taskName' type UNIQUE, 'description');"
+)
+conn.execute(stmtCreateDB)
+
+initialCredentials = [
+    ("laundry", "wash, dry, and fold clothes"),
+    ("buy groceries", "kale, feta, tomato"),
+]
+stmtInitialCred = (
+    "INSERT OR IGNORE INTO tasks (`taskName`, `description`) VALUES (?, ?);"
+)
+conn.executemany(stmtInitialCred, initialCredentials)
+conn.commit()
+conn.close()
+
+
+def tasks() -> list:
+    # connects DB
+    conn = sl.connect("tasks.db")
+    curs = conn.cursor()
+    # gets all data in table
+    curs.execute("SELECT * FROM userfoods;")
+    data = curs.fetchall()
+    return data
+
+
+@app.route("/tasks", methods=["GET"])
+def tasksList():
+    try:
+        return jsonify(meta={"message": "Success", "status": 200}, data=tasks())
+    except Exception as err:
+        return jsonify({"data": None, "meta": {"status": 400, "message": str(err)}})
+
+
+@app.route("/addTask", methods=["POST"])
```

Comment:
Since this is a HTTP API the method POST already indicates the action do theres no need to make the URL a verb; change to something like this instead:
```suggestion
@app.route("/task", methods=["POST"])
```